### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] List item: Incorrect UI Type read.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#11964]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
Table with Filer:
The list items within the 'filter' dialog are read out as ‘Link’.. Type of the UI should be read out correctly to screen reader users.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest versions

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table with Filter on toolbar. click filter icon .to open there respective dialog popup. The list items within the 'filter' dialog will be read out as ‘Link’

#### Please provide relevant source code (if applicable).
Provided stackblitz example.

#### Please provide stackblitz example(s).
https://stackblitz.com/edit/prywkj?file=package.json

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?
Reopened defect as https://github.com/SAP/fundamental-ngx/issues/11275 was closed without fix.

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**
